core/trino-main/src/test/java/io/trino/type/setdigest/TestSetDigest.java

```diff
@@ -173,4 +191,17 @@ public void testSmallLargeIntersections()
             }
         }
     }
+
+    @Test
+    public void testJaccardIndex()
```

Comment:
Move these to `io.trino.sql.query.TestSetDigestFunctions`. Also, see how the tests in that class are structure with respect to the use of `QueryAssertions` and AssertJ.